website/project/views/contributor.py

```diff
@@ -505,6 +507,29 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
     return to_addr
 
 
+@contributor_added.connect
+def notify_contributor(node, contributor, throttle=24 * 3600):
```

Comment:
Sure thing! I did write a test for throttling (although I do want to give it a better name) anything I should add to it? https://github.com/CenterForOpenScience/osf.io/pull/3385/files#diff-a4f5c4d64d4811b15e1405573c00f3f4R1754